charts/openebs/openebs-pool-disk-loss/experiment.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Kill OpenEBS pool disk loss.
+kind: ChaosExperiment
+metadata:
+  labels:
+    litmuschaos.io/name: openebs
+  name: openebs-pool-disk-loss
+  version: 0.1.0
+spec:
+  definition:
+    scope: Cluster
+    permissions:
+      - apiGroups:
+          - ""
+          - "apps"
+          - "litmuschaos.io"
+          - "batch"
+          - "extensions"
+          - "storage.k8s.io"
+          - "openebs.io"
+        resources:
+          - "pods"
+          - "jobs"
+          - "events"
+          - "pods/exec"
```

Comment:
Dont we neef pod/log too now?

Reply:
It's working without the pod/log

Reply:
Okay Got it why we need pod/log. Added the permission

Reply:
Yes we need pods/log, to get the failure step if it fails